pkg/controller/webhooks/deletionprotection_test.go

```diff
@@ -127,7 +127,7 @@ func Test_checkManagedObject(t *testing.T) {
 	assert.NoError(t, err)
 
 	// Then expect parent
-	compInfo, err = checkUnmanagedObject(context.TODO(), pvc, c, logr.Discard())
+	compInfo, err = checkUnmanagedObject(context.TODO(), pvc, c, c, logr.Discard())
```

Comment:
why do You pass the same client `c` twice?

Reply:
Because for this test, we don't use a control-cluster <-> service-cluster split. So both clients are the same.